web/src/user_topics.js

```diff
@@ -58,6 +64,62 @@ export function get_muted_topics() {
     return topics;
 }
 
+export function update_topic_visibility_policy(
+    stream_id,
+    topic,
+    new_visibility_policy,
+    from_hotkey,
+) {
+    const data = {
+        stream_id,
+        topic,
+        visibility_policy: new_visibility_policy,
+    };
+
+    channel.post({
+        url: "/json/user_topics",
+        data,
+        success() {
+            // Accidentally removing mute/unmute topic_visibility of a topic isn't as much an issue as
+            // accidentally muting/unmuting a topic, so we don't show a popup after
+            // setting topic's visibility to Inherit.
+            if (new_visibility_policy === settings_config.topic_visibility_policy.INHERIT) {
+                feedback_widget.dismiss();
+                return;
+            }
```

Comment:
I don't understand why we'd need to dismiss the feedback widget here -- is this if one toggles it back while it's still visible?

Reply:
The feedback_widget.dismiss() function was originally included in the unmute_topic function. However, upon checking it today, I realized that feedback_widget.dismiss() does the same thing as clicking on "undo mute" since it calls animate.fadeOut(). 
Therefore, I removed the if condition in the updated PR.

Reply:
> is this if one toggles it back while it's still visible?

**Yes**

@Ddharmani3 after your recent push, the widget is not dismissed on toggling back (while it's visible) (using hotkey).

Reply:
Yeah I think my comment was in error, and we actually do need this line.